cmd/cassandra-writer/main.go

```diff
@@ -106,7 +107,7 @@ func main() {
 	// Create new http server
 	httpServerConfig := server.Config{Port: defSvcHttpPort}
 
-	if err := env.Parse(&httpServerConfig, env.Options{Prefix: envPrefix, AltPrefix: envPrefixHttp}); err != nil {
+	if err := env.Parse(&httpServerConfig, env.Options{Prefix: envPrefixHttp, AltPrefix: envPrefix}); err != nil {
```

Comment:
@rodneyosodo  @dborovcanin We have planned to remove `AltPrefix` everywhere 
Can we remove in this PR

Reply:
If we have all the prefixes set correctly now, we can remove it in this PR.